app/policies/event/follow_policy.rb

```diff
@@ -3,7 +3,7 @@
 class Event
   class FollowPolicy < ApplicationPolicy
     def create?
-      user == record.user && Flipper.enabled?(:organization_announcements_tier_1_2025_07_07, record.event)
+      user == record.user
```

Comment:
For a separate PR: I think we need stricter rules on who can follow. IIRC, it's only possible to allow organizations that are either:
- transparent
- or, have at least one post
(^ i say that because that's the only time the button would be visible for following)

I think that logic probably should be located here as well.